File: C# LB Assignment/Assignment 40/program5.cs
using System;

class Test
{
public void Display(int iRow,int iCol)
{
int i=0,j=0;

for(i=1;i<=iRow;i++)
{

	for(j=1;j<=iCol;j++)
	{
	if((i>1) && (i<iRow))
	{
		if(i==j)
		{
            Console.Write(j+"\t");
		}
		else if((j==1) || (j==iCol))
		{
              Console.Write(j+"\t");
		}
		else
		{
          Console.Write("\t");
		}
		
	}
	else
	{
	 Console.Write(j+"\t");
	}

	}
	
   Console.WriteLine();
}
}


}	


class program
{
static void Main(string []arg)
{
Console.WriteLine("Enter no of row");
int row=Convert.ToInt32(Console.ReadLine());

Console.WriteLine("Enter number of col");
int col=Convert.ToInt32(Console.ReadLine());

Test obj=new Test();

obj.Display(row,col);


}	

}

File: Programming Assignment/Assignment 12/program5.c
#include<stdio.h>
void Display(int iRow,int iCol)
{
int i=0,j=0;

for(i=1;i<=iRow;i++)
{
	if(i>=2 &&i<=3)
	    {
		  for(j=1;j<=iCol;j++)
		  {
			  if(j==2 || j==3)
			  {
				  printf("*\t");
			  }
			  else
			  {
				  printf("%d\t",j);
			  }
		  }
		}
		else
		{
			for(j=1;j<=iCol;j++)
			{
				printf("%d\t",j);
			}
		}
		  printf("\n");
}
		  



}
int main()
{
	int iValue1=0,iValue2=0;
	
	printf("Enter Number of row\n");
	scanf("%d",&iValue1);
	
	printf("Enter Number of column\n");
	scanf("%d",&iValue2);
	
	Display(iValue1,iValue2);
	
	return 0;
	
	
	
}